index.js

```diff
@@ -1,42 +1,59 @@
-/* eslint-disable no-unused-vars */
+/* eslint-disable import/named */
+import BookClass from './bookClass.js';
 /* eslint-disable no-use-before-define */
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you fix your linters everywhere you disable them for best practices and clean code.

Reply:
Thank you for your feedback